lib/services/sites.js

```diff
@@ -53,8 +53,8 @@ function getSites() {
 
   files.getFolders(instanceSitesFolder).forEach(function (site) {
     var dir = path.join(instanceSitesFolder, site),
-      siteConfig = files.getYaml(path.join(dir, 'config')),
-      localConfig = files.getYaml(path.join(dir, 'local'));
+      siteConfig = files.getYamlWithEnv(path.join(dir, 'config')),
+      localConfig = files.getYamlWithEnv(path.join(dir, 'local'));
```

Comment:
Why does the sites service care that process.env is involved?  There are not env's being passed in.  There are no env's being passed out.

Encapsulation, and Single Responsibility Principle.